common/src/wdt.rs

```diff
@@ -97,5 +97,8 @@ pub fn restart_wdt(soc_ifc: &mut SocIfc) {
 /// * `soc_ifc` - SOC Interface
 ///
 pub fn stop_wdt(soc_ifc: &mut SocIfc) {
+    if !soc_ifc.debug_locked() {
+        return;
+    }
```

Comment:
Does this mean if we aren't in debug state we can never stop the wdt? I'm not sure this works for RT since it will be in low power state most of the time.

Reply:
if we are debugging we don't want to start so we don't need to stop. it is like the rom.

Reply:
Oh, I missed that it was never getting started. Thanks 😄